app/assets/main/react_components/footprint_form/OptionList.jsx

```diff
@@ -4,22 +4,27 @@ import AnswerButton from './AnswerButton.jsx';
 /**
  * Receives a list of all options for a specific question, and maps 
  */
-const OptionList = ({ onAnswerGiven, options }) => {
+const OptionList = ({ onAnswerGiven, options, selectedKey }) => {
 
   return (
       <>
         {options.map((option) => {
+          const optionObject = {
+            "key": option[0],
+            "value": option[1]
+          }
           return (
-            <div key={option[0]} className="my-3 flex-1">
+            <div key={optionObject.key} className="my-3 flex-1">
```

Comment:
Känns onödigt att introducera ett objekt som inte används utan bara refereras till
```suggestion
            <div key={option[0]} className="my-3 flex-1">
```